packages/core/src/api/guild.ts

```diff
@@ -164,6 +166,26 @@ export class GuildsAPI {
 	public async delete(guildId: Snowflake, { signal, reason }: Pick<RequestData, 'reason' | 'signal'> = {}) {
 		await this.rest.delete(Routes.guild(guildId), { reason, signal });
 	}
+	
+	/**
+	 * Add user to the guild
```

Comment:
```suggestion
	 * Adds user to the guild
```

but still poor wording

Reply:
haha ik